feature-extensions/azure/setup.py

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/env python2
+import json
+import os.path
+import subprocess
+import sys
+from collections import OrderedDict
+
+import click
+
+import git_config
+import jenkins_config
+
+
+class colors:
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
+
+
+featureName = "Azure Deployment"
+configFolder = "module_config"
```

Comment:
`installer-vars.json `is usually available in `./jazz-build-module/`. What's the purpose of this location?

Reply:
This is just a temporary folder that holds the configuration files. It is part of jazz-build-module. Would you prefer that I name it jazz-build-module?

Reply:
If this is a temp location, that should be fine.